src/components/ProgramDiv.tsx

```diff
@@ -6,12 +6,12 @@ type Props = {
 
 export const ProgramDiv = ({ time, headline, description }: Props) => {
   return (
-    <div className="flex content-start gap-10 my-6">
-      <p className="font-bold">{time}</p>
-      <div>
-        <p className="font-bold">{headline}</p>
-        <p>{description}</p>
+    <div className="flex items-start justify-start">
```

Comment:
when you set flex, the items-start and justify-start are the default values, so these are not needed here